test/metabase/api/automagic_dashboards_test.clj

```diff
@@ -171,9 +171,9 @@
 ;;; ------------------- Transforms -------------------
 
 (expect
-  [[4 1 10.0646 -165.374 "Red Medicine" 3 1 4 3 2 1]
-   [11 2 34.0996 -118.329 "Stout Burgers & Beers" 2 2 11 2 1 1]
-   [11 3 34.0406 -118.428 "The Apple Pan" 2 2 11 2 1 1]]
+  [[3 4 -165.374 10.0646 1 "Red Medicine" 1 4 3 2 1]
+   [2 11 -118.329 34.0996 2 "Stout Burgers & Beers" 2 11 2 1 1]
+   [2 11 -118.428 34.0406 3 "The Apple Pan" 2 11 2 1 1]]
```

Comment:
why are these changing?

Reply:
Just the order changed due to how we construct the `:dimensions` object (low cardinality maps in Clojure retain insertion order).